bin/runtime/src/lib.rs

```diff
@@ -256,6 +259,7 @@ impl pallet_balances::Config for Runtime {
 
 parameter_types! {
     pub const TransactionByteFee: Balance = 1;
+    pub const OperationalFeeMultiplier: u8 = 5;
```

Comment:
If I remember correctly this one is even more tricky, since it is a way how substrate node computes a fee for root calls. @pmikolajczyk41 can you maybe check on this one, since I think you worked on fee computation? Or I mixed something up? :sweat_smile:

Reply:
Can we add some comment for it then?

Reply:
done